src/config/2022.json

```
@@ -130,9 +130,7 @@
           "part": "II",
           "chapter_number": "16",
           "title": "Capabilities",
-          "slug": "capabilities",
-          "hero_dir": "2020",
-          "todo": true
+          "slug": "capabilities"
```

Can you put the `hero_dir` back? It works without it but causes a redirect:

```suggestion
          "hero_dir": "2020",
          "slug": "capabilities"
```

---

I thought that was a typo, but yea

---

Yeah not obvious, but this chapter missed the first year (2019) so only got added from 2020 onwards so needs dedicated config for that reason.